x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/index.test.tsx

```diff
@@ -0,0 +1,313 @@
+          ],
+          package: 'nginx',
+          path: 'error',
+        },
+        {
+          type: 'logs',
+          dataset: 'nginx.ingress_controller',
+          title: 'Nginx ingress_controller logs',
+          release: 'experimental',
+          ingest_pipeline: 'default',
+          streams: [
+            {
+              input: 'logfile',
+              vars: [
+                {
+                  name: 'paths',
+                  type: 'text',
+                  title: 'Paths',
+                  multi: true,
+                  required: true,
+                  show_user: true,
+                  default: ['/var/log/nginx/ingress.log*'],
+                },
+              ],
+              template_path: 'stream.yml.hbs',
+              title: 'Nginx ingress controller logs',
+              description: 'Collect Nginx ingress controller logs',
+              enabled: false,
+            },
+          ],
+          package: 'nginx',
+          path: 'ingress_controller',
+        },
+        {
+          type: 'metrics',
+          dataset: 'nginx.stubstatus',
+          title: 'Nginx stubstatus metrics',
+          release: 'experimental',
+          streams: [
+            {
+              input: 'nginx/metrics',
+              vars: [
+                {
+                  name: 'period',
+                  type: 'text',
+                  title: 'Period',
+                  multi: false,
+                  required: true,
+                  show_user: true,
+                  default: '10s',
+                },
+                {
+                  name: 'server_status_path',
+                  type: 'text',
+                  title: 'Server Status Path',
+                  multi: false,
+                  required: true,
+                  show_user: false,
+                  default: '/nginx_status',
+                },
+              ],
+              template_path: 'stream.yml.hbs',
+              title: 'Nginx stub status metrics',
+              description: 'Collect Nginx stub status metrics',
+              enabled: true,
+            },
+          ],
+          package: 'nginx',
+          path: 'stubstatus',
+        },
+      ],
+      owner: { github: 'elastic/integrations-services' },
+      latestVersion: '0.3.7',
+      removable: true,
+      status: 'not_installed',
+    },
+  } as GetInfoResponse;
+
+  const packageReadMe = `
+# Nginx Integration
+
+This integration periodically fetches metrics from [Nginx](https://nginx.org/) servers. It can parse access and error
+logs created by the HTTP server.
+
+## Compatibility
+
+The Nginx \`stubstatus\` metrics was tested with Nginx 1.9 and are expected to work with all version >= 1.9.
+The logs were tested with version 1.10.
+On Windows, the module was tested with Nginx installed from the Chocolatey repository.
+`;
+
+  http.get.mockImplementation(async (path) => {
+    if (typeof path === 'string') {
+      if (path === '/api/fleet/epm/packages/nginx-0.3.7') {
```

Comment:
FYI - as I continue to work on this test file, I think I will eventually suggest that we break out these API request test "handlers" into another test utility so that they can be used with other tests.